wdtk-datamodel/src/main/java/org/wikidata/wdtk/datamodel/json/jackson/AliasesDeserializer.java

```diff
@@ -80,8 +80,6 @@
 				}
 			}
 
-		} catch (JsonProcessingException e) {
-			e.printStackTrace();
 		} catch (Exception e) {
```

Comment:
So can this be merged? It's less clutter than before and the mentioned issue was already present.